contrib/intel/jenkins/Jenkinsfile.daily

```diff
@@ -447,39 +447,65 @@ pipeline {
                         withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin:$PYTHONPATH']) {
                           sh """
                             env
-                            (                           
+                            (
                                 cd ${env.WORKSPACE}/contrib/intel/jenkins/
                                 python3.7 runtests.py --prov=verbs --ofi_build_mode='dl'
-                                python3.7 runtests.py --prov=verbs --util=rxm --ofi_build_mode='dl'                   
-                            )  
+                                python3.7 runtests.py --prov=verbs --util=rxm --ofi_build_mode='dl'
+                            )
                          """
-                        } 
-                     }       
-       
-                 }   
+                        }
+                     }
+
+                 }
                  stage('mlx5-verbs_rxd-dl') {
                      agent {node {label 'mlx5'}}
                      steps{
                         withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin:$PYTHONPATH']) {
                           sh """
                             env
-                            (                           
+                            (
                                 cd ${env.WORKSPACE}/contrib/intel/jenkins/
-                                python3.7 runtests.py --prov=verbs --util=rxd --ofi_build_mode='dl'                   
-                            )  
+                                python3.7 runtests.py --prov=verbs --util=rxd --ofi_build_mode='dl'
+                            )
                          """
-                        } 
+                        }
                      }
-                }
-            } 
-
-   }        
-
+                 }
+                 stage('hfi1-psm3-dbg') {
```

Comment:
Is this stage name correct?

Reply:
yes, stage names are fine, they are used by Jenkins ui to indicate different stages.

Reply:
Are these the correct names?  Is psm3 linked to hfi1 and/or mlx5?

Reply:
psm3 can run on most UD capable verbs devices. PSM3 has been validated thoroughly on E810 (CVL) cards, but since you had hfi1 and mlx available I added it to them. If time is an issue, we could maybe choose one or the other until we get E810 machines to test on. That being said, this Jenkins CI job still completes quicker than the Travis CI.